### PR TITLE
MEED-200 Manage Metamask network and account switching

### PR DESCRIPTION
**Pull request by ArfaouiMohamedAmine**
Having Metamask as configured Wallet Provider, this feature will enable to handle switch of Network and account on Metamask in order to use the correct ones. In addition, a toast notif will be displayed to notify user about transaction sending status.

### CODE REVIEW

File: wallet-services/src/main/resources/locale/portlet/Portlets_en.properties [hunk 1]
```diff
@@ -72,5 +73,7 @@ exoplatform.wallet.error.wrongPrivateKey=Private key doesn't match address {0}
 exoplatform.wallet.error.errorImportingPrivateKey=Error importing private key
 exoplatform.wallet.error.errorProcessingNewKeys=Error processing new keys
 exoplatform.wallet.error.walletNotFound=Can't find your wallet
+
 exoplatform.wallet.metamask.welcomeMessage=Welcome to digital workplace \n\nClick to SignIn and accept digital workplace terms. This actions will not trigger a blockchain transaction and or any Gas fees. \n\nWallet address: \n{0} \n\nNonce: \n{1}
 exoplatform.wallet.metamask.message.connectedSuccess=Wallet connected to Metamask successfully !
+exoplatform.wallet.metamask.message.transactionSent=Tranasction sent
```
Comment (boubaker) [inline wallet-services/src/main/resources/locale/portlet/Portlets_en.properties]: add this to Wallet_en.properties please.

File: wallet-services/src/main/resources/locale/portlet/Portlets_en.properties [hunk 2]
```diff
@@ -23,6 +23,7 @@ exoplatform.wallet.settings.meedsWallet=Meeds Wallet
 exoplatform.wallet.settings.useMetamask=Use Metamask
 exoplatform.wallet.settings.metamaskInstallation=Add Metamask add-on to your browser through this link
 
+exoplatform.wallet.message.followTransaction={0} link to follow your transaction
```
Comment (boubaker) [inline wallet-services/src/main/resources/locale/portlet/Portlets_en.properties]: add this to Wallet_en.properties please.

File: wallet-webapps-common/src/main/webapp/vue-app/js/TokenUtils.js [hunk 3]
```diff
@@ -238,6 +238,7 @@ export function getTransactionCount(walletAddress, status) {
   return window.localWeb3.eth.getTransactionCount(walletAddress, status || 'latest');
 }
 
+
```
Comment (boubaker) [inline wallet-webapps-common/src/main/webapp/vue-app/js/TokenUtils.js]: to remove

File: wallet-services/src/main/resources/locale/portlet/Portlets_en.properties (hunk 1 above)
Comment (ArfaouiMohamedAmine) [reply wallet-services/src/main/resources/locale/portlet/Portlets_en.properties]: done

File: wallet-services/src/main/resources/locale/portlet/Portlets_en.properties (hunk 2 above)
Comment (ArfaouiMohamedAmine) [reply wallet-services/src/main/resources/locale/portlet/Portlets_en.properties]: done

File: wallet-webapps-common/src/main/webapp/vue-app/js/TokenUtils.js (hunk 3 above)
Comment (ArfaouiMohamedAmine) [reply wallet-webapps-common/src/main/webapp/vue-app/js/TokenUtils.js]: done !

File: wallet-services/src/main/resources/locale/portlet/Portlets_en.properties [hunk 4]
```diff
@@ -72,5 +72,6 @@ exoplatform.wallet.error.wrongPrivateKey=Private key doesn't match address {0}
 exoplatform.wallet.error.errorImportingPrivateKey=Error importing private key
 exoplatform.wallet.error.errorProcessingNewKeys=Error processing new keys
 exoplatform.wallet.error.walletNotFound=Can't find your wallet
+
```
Comment (boubaker) [inline wallet-services/src/main/resources/locale/portlet/Portlets_en.properties]:
```suggestion
```

**Review by boubaker — APPROVED** [review]

**Pull request merged**